GitHubLabels.ps1

```diff
@@ -1104,16 +1140,26 @@ function Set-GitHubIssueLabel
             'labels' = $labelNames
         }
 
+        $shouldProcessMessage = "Set GitHub Issue Label(s) $($Label -join ', ')"
+
+        if ([System.String]::IsNullOrEmpty($Label))
+        {
+            $ConfirmPreference = 'Low'
```

Comment:
You wanted a `Confirm` prompt to be shown if all labels were removed. (i.e. `$Label` is null or empty.) but not otherwise.

`ConfirmPreference` defaults to `High` and `ConfirmImpact` defaults to `Medium` and a `Confirm` prompt is only shown when the `ConfirmImpact` argument is equal to or greater than the value of the `ConfirmPreference` variable, so setting `ConfirmPreference` to `Low` or `Medium` will cause the `Confirm` prompt to be shown.